Frontend/ui-customer/src/Components/IndividualRideCard/View.purs

```diff
@@ -365,14 +365,15 @@ viewDetailsAndRepeatRide push state =
       , height WRAP_CONTENT
       , text $ getString VIEW_DETAILS
       , color Color.blue900
-      , padding $ Padding 26 18 50 3
+      , if state.isSrcServiceable then padding $ Padding 26 18 50 3 else padding $ Padding 26 18 26 3
```

Comment:
We already have a variable with which we were handling this case and were showing the full screen popUp, can we use the same one?

Reply:
I didn't get it, on home screen we were showing popup using `showlocUnserviceablePopUp`